CMakeLists.txt

```diff
@@ -168,9 +174,12 @@ find_package(PythonInterp 3.8 REQUIRED)
 cpp_cc_strip_python_shims(EXECUTABLE "${PYTHON_EXECUTABLE}" OUTPUT PYTHON_EXECUTABLE)
 include(cmake/hpc-coding-conventions/cpp/cmake/bbp-find-python-module.cmake)
 cpp_cc_find_python_module(jinja2 2.9.3 REQUIRED)
-cpp_cc_find_python_module(pytest 3.3.0 REQUIRED)
-cpp_cc_find_python_module(sympy 1.3 REQUIRED)
-cpp_cc_find_python_module(textwrap 0.9 REQUIRED)
+if(NMODL_ENABLE_TESTS)
+  cpp_cc_find_python_module(pytest 3.3.0 REQUIRED)
+endif()
+if(NOT NMODL_BUILD_WHEEL)
+  cpp_cc_find_python_module(sympy 1.3 REQUIRED)
```

Comment:
I guess they're split into build, run and test dependencies? If yes shuffle and please add a comment.

Alternatively, should we be checking these things in a CMakeLists.txt or can we all checks?

Reply:
CMake doesn't make the distinction between build, run, or other dependencies, so I would be in favor of removing all of the checks here since `pip install -r requirements.txt` installs all of them anyway.

Reply:
Removed discovery of all Python packages in CMakeLists.txt now so we don't need to keep track of the info in two places.